### ISSUE TITLE
Typing error on 2.6.84

### ISSUE THREAD

**Issue by EdoardoTona** [gildas-lormeau/zip.js#408]
With the version 2.6.84 I get this error:

```
Error: node_modules/@zip.js/zip.js/index.d.ts:873:18 - error TS2430: Interface 'Entry' incorrectly extends interface 'EntryMetaData'.
  The types returned by 'getData(...)' are incompatible between these types.
    Type 'Promise<Type | undefined>' is not assignable to type 'Promise<Type>'.
      Type 'Type | undefined' is not assignable to type 'Type'.
        'Type' could be instantiated with an arbitrary type which could be unrelated to 'Type | undefined'.

873 export interface Entry extends EntryMetaData {
                     ~~~~~
```

moreover I get other errors in my code because the `Entry.getData()` returns `Blob | undefined` instead of `Blob` as before (maybe this is the expected behaviour but it's a breaking change for a minor update)

**Comment by gildas-lormeau** [comment]
Thank you, I just published the version `2.7.0` which should fix this issue. I did not considered as a breaking change because the library is written in JS.

**Comment by gildas-lormeau** [comment]
The version `2.7.1` fixes the breaking change.
